### DISCUSSION TITLE
请问如果我想指定新的link，url和cite超链接的颜色应该怎么做呢？

### DISCUSSION THREAD

**Discussion by ODYYYYYY** [stone-zeng/fduthesis#336] [Q&A]
我尝试直接在dtx中第5927行增加一个颜色组，但是xelatex编译main.tex文件后会报错。
![image](https://github.com/stone-zeng/fduthesis/assets/110674923/e4f70f8d-a454-4be5-9384-1be1297aa516)

**Comment by SwitWu** [comment]
你只编辑 dtx 没用，需要改 `fduthesis.cls`

**Reply by ODYYYYYY** [reply]
感谢回复，不过下载的zip文件中似乎没找到cls文件，这边已经用hypersetup进行覆盖设置。

**Reply by xkwxdyy** [reply]
你不要下 source code，去 release 下载。
